### ISSUE TITLE
Publicly viewable page with results

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/budgetparty#215]
A budget party judge, elected official, or member of the public should be able to see past results of a Budget Party session in a specialized view. You should be able to navigate to this view page either from the homepage or via the navigation. It should live at a route like `/results` or `/public`.

The results would be retrieved via a query to our firebase backend as a service. We might want to filter the response data by a date range or at least sort by the most recent submissions and paginate backwards in time.

Table layout might be the easiest but there are probably better ways to visualize the results from different teams.

**Comment by carlvlewis** [comment]
Once we grab the latest commits for our planned deployment, @mateoclarke, the current table setup you have at `/results` we're planning on adding in some d3.js visualizations using crossfilter.js and dc.js.

**Comment by carlvlewis** [comment]
Not sure what we changed to fix this, but now we do indeed have the submission data being saved in Firebase!  🎉 Yay! Thanks, @mateoclarke! Now on to getting that submission data out of Firebase as an API to power some public-facing visualizations. 💯 

Noticed the `/results` public-facing page had stopped working at austinbudget.party/results, though it was working the other day. Appears to be a permissions error, mayhaps? Not sure if that's a bug or a temporary perms change. Just letting you all know!

We've got a few customizations on the roster but excited about getting this into the wild soon! Having one large-gathering 'official' Budget Party at https://citycampsav.org Oct. 20, then distributed BudgetParty nights over the next three weeks.
